### DISCUSSION TITLE
Add NusantaraProject-ROM-12.1

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#15292] [General]
https://github.com/NusantaraProject-ROM/android_manifest/tree/12.1

Plz consider to add it because it is high chosen gaming rom..

**Comment by Apon77** [comment]
https://github.com/ROM-builders/temporary/commit/d89272a81a71176f8e6ddc612f38641f98855c82

**Reply by ghost** [reply]
Is the source close!? 

Very strange!!

Check maybe it open
